apps/console/src/features/applications/components/forms/inbound-saml-form.tsx

```diff
@@ -890,36 +906,40 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
                                 </Hint>
                             </Grid.Column>
                         </Grid.Row>
-                        <Grid.Row columns={ 1 }>
-                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
-                                <Field
-                                    ref={ signatureValidationForArtifactBinding }
-                                    name="signatureValidationForArtifactBinding"
-                                    label=""
-                                    required={ false }
-                                    requiredErrorMessage="this is needed"
-                                    type="checkbox"
-                                    value={
-                                        isArtifactBindingEnabled && initialValues?.singleSignOnProfile
-                                            .enableSignatureValidationForArtifactBinding ?
-                                            ["enableSignatureValidationForArtifactBinding"] : [] }
-                                    children={ [
-                                        {
-                                            label: t("console:develop.features.applications.forms.inboundSAML" +
-                                                ".sections.ssoProfile.fields.artifactBinding.label"),
-                                            value: "enableSignatureValidationForArtifactBinding"
-                                        }
-                                    ] }
-                                    readOnly={ readOnly }
-                                    disabled={ !isArtifactBindingEnabled }
-                                    data-testid={ `${ testId }-artifact-binding-signature-validation-checkbox` }
-                                />
-                                <Hint disabled={ !isArtifactBindingEnabled }>
-                                   { t("console:develop.features.applications.forms.inboundSAML.sections" +
-                                           ".ssoProfile.fields.artifactBinding.hint") }
-                                </Hint>
-                            </Grid.Column>
-                        </Grid.Row>
+                        {isArtifactBindingAllowed
```

Comment:
Lets wrap the multiline JSX with `()`
```suggestion
                        {
                            isArtifactBindingAllowed
                                ? (
                                    JSX
                                )
                                : null
```

Reply:
Addressed with aaa965ccd1f35f4c6c2c998bf202ab99a9f7d5ef